src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import { FilePlus, FolderPlus, Home, Share2, Files } from "lucide-react";
+
+export function Sidebar() {
+  return (
+    <div className="col-span-2 row-span-5 h-full bg-gray-200">
```

Comment:
There are a lot of `<div>` elements in this file. Since this is an aside navigation, you can use the `<aside>` tag. Also, try to remove all the unnecessary `<div>` elements.